antidox/perspective.py

```diff
@@ -110,35 +111,31 @@ def contains_toxicity(perspective_response):
   return is_toxic
 
 
-def use_query(client, sql_query):
+def use_query(args.content, sql_query):
```

Comment:
I think using args.content this way won't work, you will just want to make this input arg a string variable

Reply:
done